recipes/gatk4/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 source:
   fn: gatk-{{ version }}.zip
   url: https://github.com/broadinstitute/gatk/releases/download/{{ version }}/gatk-{{ version }}.zip
-  md5: 87b173e3abde88de2d9fcb601b4902e8
+  md5: 1b4c7ade9aa3740a3adde6922ec35ca0
```

Comment:
We should update this to `sha256` at some point. Do you want to do this now?

Reply:
That'd be
```yaml
  sha256: 11abae169b73946d52e7831d8ada8b01ab05c517bb60abbf4046eeed3c696065
```

Reply:
Didn't see that before it was too late. Next time I'm making a pull request, I will make sure to replace MD5s by SHA256s.